src/configs/kafka.js

```diff
@@ -47,6 +47,7 @@ module.exports = async () => {
 			},
 		})
 	}
+
```

Comment:
This file is not updated in this PR right? can we remove this additional line? @priyanka-TL